proto/proxy/destination/destination.proto

```diff
@@ -75,11 +75,28 @@ message WeightedAddrSet {
 }
 
 message WeightedAddr {
-  common.TcpAddress addr = 1;
-  uint32 weight = 3;
-  map<string, string> metric_labels = 4;
+    common.TcpAddress addr = 1;
```

Comment:
I realize the indentation in this file was already inconsistent, but the bulk of it was using 2-space indents, while your change uses 4. Can I bug you to convert this whole file to 2-space indents? That should only require changes from lines 78 to 101.

Reply:
No problem, sorry about that!